api/client.yaml

```diff
@@ -1513,6 +1513,249 @@ paths:
             application/json:
               schema:
                 $ref: 'https://raw.githubusercontent.com/moov-io/base/master/api/common.yaml#/components/schemas/Error'
+  /customers/{customerID}/representatives:
+    post:
+      tags: [Customers]
+      summary: Add customer representative
+      description: Add a Customer Representative
+      operationId: addCustomerRepresentative
+      parameters:
+        - name: X-Request-ID
+          in: header
+          description: Optional requestID allows application developer to trace requests through the systems logs
+          example: rs4f9915
+          required: true
+          schema:
+            type: string
+        - name: X-Organization
+          in: header
+          description: Value used to separate and identify models
+          example: org342
+          schema:
+            type: string
+        - name: customerID
+          in: path
+          description: customerID of the Customer for whom to add the representative
+          required: true
+          schema:
+            type: string
+            example: e210a9d6-d755-4455-9bd2-9577ea7e1081
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateCustomerRepresentative'
+      responses:
+        '200':
+          description: A customer object
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Customer'
+        '400':
+          description: Customer representative was not added, see error(s)
+          content:
+            application/json:
+              schema:
+                $ref: 'https://raw.githubusercontent.com/moov-io/base/master/api/common.yaml#/components/schemas/Error'
+  /customers/{customerID}/representatives/{representativeID}:
+    put:
+      tags: [Customers]
+      summary: Update customer representative
```

Comment:
Can we trim these summaries up? They're what populates this field on api.moov.io/v1/ 

```suggestion
      summary: Update Customer Representative
```

![Screen Shot 2020-11-02 at 2 39 38 PM](https://user-images.githubusercontent.com/120951/98175201-f9d9b480-1eaa-11eb-95dc-40f627db51c5.png)